packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -54,6 +54,12 @@ class FlutterCommandResult {
   final DateTime endTimeOverride;
 }
 
+/// Common flutter command line options.
+class FlutterOptions {
+  static const String kExtraFrontEndOptions = 'extra-front-end-options';
+  static const String kExtraGenSnapshotOptions = 'extra-gen-snapshot-options';
```

Comment:
These could be declared top-level, so you wouldn't need the `FlutterOptions.` prefix. Your call.
Would it make sense to add `preview-dart-2` too?

Reply:
If they're not global you can probably remove the `k` prefix. The prefix is mostly about making constants group together in documentation, autocomplete, and the like.